data.tf

```diff
@@ -0,0 +1,23 @@
+module "ssm_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+
+  delimiter = "/"
+
+  context = module.this.context
+}
```

Comment:
Does it make sense to have a `slash` label? Because it is the same as the autoscaling one 🤔

Reply:
I don't think it makes sense, because both serve a different purpose.